src/platforms/zephyr/src/main.c

```diff
@@ -0,0 +1,134 @@
+
+/*
+ * This file is part of AtomVM.
+ *
+ * Copyright 2023 Winford (Uncle Grumpy) <winford@object.stream>
```

Comment:
If this `main.c` is based on some other existing `main.c`, let's report original authors too.